posts/round7_FR.md

```diff
@@ -0,0 +1,106 @@
+[categorie] : <> (Général)
+[date] : <> (2020/10/18)
+[titre] : <> ( 7ème tour des subventions Gitcoin - Rétrospective )
+[pandoc] : <> (—mathjax)
+
+# 7ème tour des subventions Gitcoin - Rétrospective
+
+Le septième tour des subventions Gitcoin s'est terminé avec succès! Ce tour a connu une augmentation de l'intérêt pour Gitcoin sans précédent, ce qui s'est traduit par 274.830 $ de contributions et 450.000 $ de fonds de contrepartie répartis sur 857 projets.
+
+La structure des catégories a été une fois de plus modifiée; cette fois-ci, divisé entre "tech DApp", "infrastructure" et "communauté". Voici les résultats :
```

Comment:
`La structure des catégories a été une fois de plus modifiée; cette fois-ci, divisé entre "tech DApp", "infrastructure" et "communauté". Voici les résultats :`
La structure des catégories a été une fois de plus modifiée; cette fois-ci, divisée entre "tech DApp", "infrastructure" et "communauté". Voici les résultats :

Reply:
"divisé" takes the gender of "structure", juste as "modifié**e** "
=> **divisée**